### PR TITLE
DAS-1935 - Enable RSSMIF16D production HOSS regression tests.

### PR DESCRIPTION
**Pull request by owenlittlejohns**
## Description

This PR updates the HOSS notebook so that the RSSMIF16D regression tests can be run against production. This collection is now available in production, and GHRC has associated it with HOSS. However, the GES DISC collections in the notebook still aren't configured in HOSS, so I've made the decision to run tests a little less global.

## Jira Issue ID

DAS-1935 (not an A/C, but discovered while testing)

## Local Test Steps

* Pull this branch.
* Run the notebook against UAT and Production, by setting the appropriate `HARMONY_HOST_URL`.
* Note that the requests that use `hoss_info['collection']` run and pass, but that other requests are skipped.
* **Note - I've seen some slightly slow Harmony performance in production, where requests have been queued for a while before executing, but the requests were passing when they eventually ran.**

## PR Acceptance Checklist
* ~~Acceptance criteria met~~
* [x] Tests added/updated (if needed) and passing
* ~~Documentation updated (if needed)~~

### CODE REVIEW

File: test/hoss/HOSS_Regression.ipynb
Comment (owenlittlejohns) [inline test/hoss/HOSS_Regression.ipynb:151]: This is exactly the same granule as used in UAT, so the same reference files are valid.

File: test/hoss/version.txt
Comment (owenlittlejohns) [inline test/hoss/version.txt:1]: Wasn't really sure on the best type of version to bump. I don't think it's a major version bump, but could be convinced to make this a minor version bump.

**Comment by owenlittlejohns** [comment]
I've been chatting with Chris Durbin about the tasks getting stuck in a Queued status. It's because someone set a couple of 2000 granule requests going for GHRSST MUR data in the last couple of days. The individual invocations of HOSS for GHRSST MUR are slow, and they are sucking up resources and so the requests in this PR are getting stuck behind these others.

TL;DR: Any slow performance when testing this notebook against production are likely due to Harmony trying to run a couple of other _massive_ requests over the next couple of days.

File: test/hoss/version.txt
Comment (flamingbear) [reply test/hoss/version.txt:1]: No strong opinions, so this is fine with me.

**Review by flamingbear — APPROVED** [review]
I'm going to approve this, even though I did not get it to run all of the cells in PROD.  It has taken all day, each cell ~45 min plus the disconnects and timeouts.  I did see that the cells are run and successful when configured with `collection` and that the cell is skipped when configured with `temporal_collection`.  Everything else checks out.

**Comment by owenlittlejohns** [comment]
Thanks, Matt. I'll hold off until those _really_ big requests are finished in production (in a couple of days!), and just quickly breeze through the notebook to be sure, before merging this in.

**Comment by flamingbear** [comment]
that's fine, I know I'm just out for a few days and didn't want this to get hung up by that.

**Pull request merged**
